packages/angular-cli/tasks/server/universal-dev-server.ts

```diff
@@ -0,0 +1,64 @@
+import * as path from 'path';
+const nodemon = require('nodemon');
+const tinyLr = require('tiny-lr');
+
+export class UniversalDevServer {
+  private appPath: string = '';
+  private nodemonConfig: any;
+  private demon: any;
+  private liveServer: any;
+
+  constructor(private webpackCompiler: any, private webpackDevServerConfiguration: any) {
```

Comment:
I am not familiar with the webpack-dev-middleware. Is it possible to reload the whole server code when somethings change? Because in universal the `server.ts` file is part of the business logic and for DX it must reload the server code when the user changes something. I'll have a look at it. maybe we can chat or something?